components/components-style.css

```diff
@@ -115,3 +117,188 @@ a {
 .item-text {
   color: var(--font-color);
 }
+/** Cards **/
+.cards {
+  margin-left: 5rem;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.card {
+  margin: 2rem;
+  max-width: max-content;
+  height: fit-content;
+  display: flex;
+  padding: 0.5rem;
+  flex-direction: column;
+  justify-content: flex-start;
+  flex-wrap: wrap;
+  z-index: 3;
+  background-color: var(--white);
+  gap: 0.5rem;
+  border-radius: 1rem;
+  color: var(--font-color-black);
+}
+img {
+  border-radius: 1rem;
+}
+.btn {
+  background-color: var(--primary-color);
+  color: white;
+  border-radius: 1rem;
+  font-size: large;
+  padding: 0.5rem;
+}
+.btn:hover {
+  background-color: var(--card-btn);
+}
+
+.code {
+  margin: 2rem;
+  border-radius: 1rem;
+  overflow-x: auto;
+}
+
+.heading {
+  font-size: xx-large;
+  text-align: center;
+}
+
+.card-div {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  /* padding: 2rem; */
+  /* border:0.5rem solid var(--font-color); */
+  /* border-radius: 1rem; */
+  margin: 2rem;
+}
+
+.card-basic:hover {
+  box-shadow: 0 14px 28px rgba(243, 238, 238, 0.25),
+    0 10px 10px rgba(241, 218, 218, 0.22);
+}
+
+.horizontal {
+  width: fit-content;
+  max-height: fit-content;
+  display: flex;
+  flex-direction: row;
+}
+.card-items-horizontal {
+  row-gap: 1rem;
+  display: flex;
+  flex-direction: column;
+}
+/** Text only Card **/
+.card-items-text {
+  display: flex;
+  flex-direction: column;
+  gap: 1.5rem;
+}
+.text-only-font {
+  color: var(--card-btn);
+}
+
+/** Badge Card **/
+.badges{
+  padding-top: 2rem;
+  position: relative;
+}
+.btn-badge {
+  position: relative;
+}
+.badge {
+  position: absolute;
+  z-index: 111;
+  height: 1rem;
+  width: 1rem;
+  border-radius: 50%;
+  background-color: var(--green);
+  top: 1.3rem;
+}
+
+.text-badge{
+  position: absolute;
+  width:4rem;
+  height: 1.5rem;
+  text-align:center;
+  background-color: var(--card-btn);
+  padding:-.5rem;
+  border-radius: 0.5rem;
+  color: var(--white);
+  z-index: 111;
+  top: 0.5rem;
+  left:0.2rem;
+  
+}
+
+.badge.red {
+  background-color: var(--red);
+}
+.badge.blue {
+  background-color: var(--card-btn);
+}
+
+/** Cards with Dismiss **/
+
+.dismiss-btn {
+  margin-top: 1rem;
+  display: flex;
+  justify-content: flex-end;
+  gap: 1rem;
+}
+.btn.red {
+  background-color: var(--red);
+}
+.btn.blue {
+  background-color: var(--card-btn);
+}
```

Comment:
`.btn.red` `.badge.red` instead of having two separate rules that define the same thing combine them into one. `.red` or `background-red` can be defined and used wherever necessary

Reply:
Similarly, for blue